Automattic/woocommerce-connect-client
# Label Rates UI Step

Fixes #506.

Does not implement:
* Selecting a "similar" rate if customer-chosen rate isn't available.
* Price display options (no API equivalent discovered yet)

To test:
* Start "create label" flow
* If Origin and Destination are already OK, Rates step should show spinner upon opening modal
* Otherwise, once Origin and Destination are "green", Rates should fetch
* Change either Origin or Destination address, verify that Rates refresh
* Verify that when customer-chosen rate isn't available, notice is shown
  * Easiest way is to choose Priority X-Day shipping at checkout, and change one address to be farther away, such that it's now Priority Y-Day that's available
